src/main/java/hw3/components/RadioButtons.java

```diff
@@ -0,0 +1,30 @@
+package hw3.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import java.util.List;
+
+public class RadioButtons {
```

Comment:
What are these radio buttons? 
I suppose that this class will not work for the others similar elements, except specified here

Reply:
Renamed class